packages/@atjson/source-html/src/converter/third-party-embeds.ts

```diff
@@ -7,9 +7,7 @@ function isCerosExperienceFrame(a: Annotation<any>) {
 
 function isCerosOriginDomainsScript(a: Annotation<any>) {
   return (
-    a.type === "script" &&
-    a.attributes.dataset &&
-    a.attributes.dataset["ceros-origin-domains"] != null
+    a.type === "script" && a.attributes.src?.indexOf("view.ceros.com") !== -1
```

Comment:
Not a blocker, but I feel like it's worth it to parse the src attribute as a URL and verify the host. Probably won't be meaningfully different in many cases at all, but can't see a compelling reason to not do it.